dwr-hecdssvue/src/main/java/gov/ca/dwr/hecdssvue/views/DSSFileView.java

```diff
@@ -511,59 +567,13 @@ public void saveProject(String fn){
 	
 	public void selectFiles(){
 		selectFileNames();
-		//DataOps.setProject();
```

Comment:
@hxiedwr - this commented out line is similar to what we are hitting. Why was this line commented out and do we even need to call it in the DTSView.createPartControl anymore?

Reply:
https://github.com/CentralValleyModeling/wrims/commit/428f5098dc81ae91a3ed153444120ead3d65a012

Reply:
I created issue: https://github.com/CentralValleyModeling/wrims/issues/178